src/modules.js

```diff
@@ -113,6 +113,7 @@ var LibraryManager = {
         'library_fs.js',
         'library_memfs.js',
         'library_tty.js',
+        'library_pipefs.js',
```

Comment:
It is likely that these names go to a dictionary later on, so the alphabetical ordering will be restored later even if they are not present now. However I think no need to worry if it works, since issues of not having initialized the filesystem should be apparent if that happens. (For curiosity, you could try renaming the file to `library_aipefs.js` or something similar and see if that causes issues)